test/unit/processing/get-payment-requests-from-file.test.js

```diff
@@ -0,0 +1,224 @@
+jest.mock('../../../app/processing/siti-agri/build-payment-requests')
```

Comment:
`validate-sequence.js` needs unit tested

Reply:
Agree.  Added tests covering all behaviour with the exception of the disabling of validation.  Having some issues with mocking the configuration for those ones so will add later.